src/api_server/src/parsed_request.rs

```diff
@@ -165,11 +166,13 @@ pub enum Error {
     InvalidPathMethod(String, Method),
     // An error occurred when deserializing the json body of a request.
     SerdeJson(serde_json::Error),
+    // An error occured while processing mmds configuration request.
+    MmdsConfig(MmdsConfigError),
```

Comment:
`MmdsConfig()` seems rather a very specific error for this enum, which should contain only generic errors for parsing requests.